### PR TITLE
Darya - Guess Who? 

### PR DESCRIPTION
**Pull request by DALA746**

### CODE REVIEW

File: code/confetti.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
+
+// THIS CODE I DONT QUIT UNDERSTAND BUT IT MAKES CONFETTI RAIN :)
+
+
```
Comment (Jsfrulle) [inline code/confetti.js:4]: Love that you use this effect 👯

File: code/confetti.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+		particle.tiltAngleIncrement = Math.random() * 0.07 + 0.05;
+		particle.tiltAngle = Math.random() * Math.PI;
+		return particle;
+	}
+
+	function toggleConfettiPause() {
+		if (pause)
+			resumeConfetti();
+		else
+			pauseConfetti();
```
Comment (Jsfrulle) [inline code/confetti.js:60]: if you want to pause the confetti can you call pauseConfetti()?

File: code/script.js [hunk 3]
```diff
@@ -1,7 +1,24 @@
...
+const playAgainButton = document.getElementById('playAgain')
+const startButton = document.getElementById('start')
+const filter = document.getElementById('filter')
+const startWindow = document.getElementById('start-window')
+
+const startAudio = document.getElementById('audio')
+const winningAudio = document.getElementById('audio-winning')
+const lostAudio = document.getElementById('audio-lost')
+
+const modal = document.getElementById("myModal");
```
Comment (Jsfrulle) [inline code/script.js:13]: maybe use a more describing name of this variable

File: code/script.js [hunk 4]
```diff
@@ -221,105 +238,171 @@ const generateBoard = () => {
...
-
-    for accessories and other
-      charactersInPlay = charactersInPlay.filter((person) => person[category].includes(value))
-      or
-      charactersInPlay = charactersInPlay.filter((person) => !person[category].includes(value))
-  */
-
-  // Invoke a function to redraw the board with the remaining people.
+  const { category, value } = currentQuestion // taking value from category and value
+  modalWindow(category, value, keep) // calling this function and passing 3 paramenters, this function filter and showing message to the user
```
Comment (Jsfrulle) [inline code/script.js:348]: smart! :D

File: code/style.css [hunk 5]
```diff
@@ -145,6 +184,31 @@ button {
...
+.start-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: 700px;
+  text-align: center;
+}
+
+
```
Comment (Jsfrulle) [inline code/style.css:211]: Dayra You did the game like I wish I had done mine SO IMPRESSIVE!!! I love the confetti, starter window, and custom popup!!!!! the only thing I think you could think about is to name the variables/tags to more describing - like starter window (if not the modal window is the right description for this variable and I just don't know that :) ) Good work!!! - -[ YES]  Am I able to understand the code easily?
- [NO ]  Does the code have lots of duplicates? Could it be tidied up?
- [ NO]  Does it work or does anything look broken?
- [YES]  Does it follow the general and blue requirements?

File: code/confetti.js (hunk 1 above)
Comment (DALA746) [reply code/confetti.js:4]: Thanks Jessica! Yeah, really nice effect, but a lot of code that I don't understand :)

File: code/confetti.js (hunk 2 above)
Comment (DALA746) [reply code/confetti.js:60]: Yes, I think so! It sounds logical. I will try to manipulate with that code :)

File: code/script.js (hunk 3 above)
Comment (DALA746) [reply code/script.js:13]: I must be better to name classes, id and variables :) but sometimes it is hard. Thanks I will think of that in my future projects!

File: code/script.js (hunk 4 above)
Comment (DALA746) [reply code/script.js:348]: Those functions can be a struggle, but when they work,  your code can be so much shorter and more easy to read :)

File: code/style.css (hunk 5 above)
Comment (DALA746) [reply code/style.css:211]: Thank you Jessica for your code review! :) Really great inputs that I will apply for my next projects :)

**Pull request closed without merging**